CODEOWNERS

```diff
@@ -347,6 +347,7 @@
 /subsys/fs/littlefs_fs.c                  @pabigot
 /subsys/fs/nvs/                           @Laczen
 /subsys/logging/                          @nordic-krch
+/subsys/logging/log_backend_net.c         @jukkar
```

Comment:
do you want to have nordic-krch also be a codeowner of this file or only you?

Reply:
I am just trying to setup the system so that I am put as a reviewer if someone touches that file (in order to avoid partially wrong fix introduced by #19637 to this file). So nordic-krch should be also codeowner and he is marked so for the whole logging subdirectory, or do we need to place his name here to this line too?

Reply:
>  or do we need to place his name here to this line too?

Yep, you need to. This is a "last match wins" system. So if the last line was 
`* @aescolar` I would be the *only* codeowner of the whole tree.

Reply:
Ok, I will send a new version.